src/registry.ts

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/* eslint-disable no-underscore-dangle */
```

Comment:
Ideally, I'd prefer to not disable any of the eslint rules

Reply:
👍